Makefile.docker

```diff
@@ -14,9 +14,9 @@
 #
 # Testing docker is done e.g. via:
 #
-#  export DOCKER_PASSWORD=<pass>
```

Comment:
I actually think this space was intentional, so that `DOCKER_PASSWORD` would not get stored in the shell history..